cont3xt/vueapp/src/App.vue

```diff
@@ -37,6 +37,18 @@
           <br>
           <code>'>'</code> - toggle the link group panel
           <br>
+          <code>'-'</code> - collapse all top-level indicator result tree nodes
```

Comment:
this should be shift-'-' right? since an actually won't work :)

Reply:
yes, so 'shift -', then? do we want 'shift +' to match that, or keep just '+' since you need shift for that anyways?

Reply:
sure you can make it shift + so it looks pretty next to it :)